src/Diagnostics.HealthChecks/Microsoft.Omex.Extensions.Diagnostics.HealthChecks.csproj

```diff
@@ -20,6 +20,9 @@
     <PackageReference Include="Newtonsoft.Json" /> <!-- Bump version of transitive dependency -->
     <PackageReference Include="System.ComponentModel.Annotations" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Condition="'$(TargetFramework)' != '$(DefaultNetCoreVersion)'" Include="Microsoft.Bcl.AsyncInterfaces" />
```

Comment:
Any chance healthcheck project could stop target netstandard2.0 I assumed Omex does not have any services that target full framework anymore?

Reply:
Services target net6 so they get netstandard20 version of dotnet package. We can't upgrade to net7 yet.

Reply:
Maybe leaving net6.0 and net7.0 would be easier that still supporting netstandard2.0 or at least removing them after full move to net7.0.